opm/material/fluidmatrixinteractions/EclEpsTwoPhaseLawParams.hpp

```diff
@@ -1,4 +1,4 @@
-// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
+
```

Comment:
Is this bit intentional or a left-over from local development?

Reply:
ide artifact. fixed.